### PR TITLE
fix: Windows install instructions

### PR DESCRIPTION
**Pull request by paulswartz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,18 +17,18 @@ $ brew install proj freetds
...
+$ pipenv sync  # --dev if you want to do development on the scripts
 $ cp .env.template .env
 $ vi .env  # configure environment variable
 ```
 
 ### Windows
 ```
 $ choco install pyenv-win
-$ pyenv install
+$ pyenv install 3.9.2
```
Comment (dks-mbta) [inline README.md:28]: This gives an error: 
 pyenv-install: definition not found: 3.9.2

The latest I see is 3.9.0a4 .

**Review by dks-mbta — COMMENTED** [review]
After installing the older pyenv, I get this error running hastus_sync:

ModuleNotFoundError: No module named 'smbclient'

File: README.md (hunk 1 above)
Comment (paulswartz) [reply README.md:28]: That sounds like your pyenv is out of date: try `pyenv update`?

**Comment by paulswartz** [comment]
@dks-mbta can you post (either here or in Slack) the output from all the installation commands?

**Pull request merged**